tests/ui/manual_memmove.stderr

```diff
@@ -0,0 +1,22 @@
+error: it looks like you're manually copying within a slice
+  --> $DIR/manual_memmove.rs:8:5
+   |
+LL | /     for i in 0..(arr.len() - 4) {
+LL | |         // left shift by 4
+LL | |         arr[i] = arr[i + 4];
+LL | |     }
+   | |_____^ help: try replacing the loop by: `arr.copy_within(4..((arr.len() - 4) + 4), 0);`
```

Comment:
The range bound could be simplified. My suspicion is that MinifyingSugg doesn't work properly here, but I would be "fine" with this output.